config/routes.yaml

```diff
@@ -1,5 +1,2 @@
-#index:
-#    path: /
-#    controller: App\Controller\DefaultController::index
 gssp_saml:
```

Comment:
Ik heb routing vervangen in SelfService door #[Route()] attribute

Reply:
This opened a can of worms ;)

I already defined the attribute based routes everywhere. But the app is not loading them appropriately. Thats why all the extra config is added. I can not figure out how to configure this correctly. I'm putting this on the backlog for now. But a thing to fix for surre!